erpcgen/src/templates/c_coders.template

```diff
@@ -228,7 +244,15 @@ codec->readData({$info.name}, {$info.sizeTemp} * sizeof({$info.builtinTypeName})
 {# Encode sending data #}
 {% def encodeBuiltinType(info) -----------------  %}
 {% if info.builtinType == "kStringType" %}
-codec->writeString(strlen((const char*){$info.name}), (const char*){$info.name});
+{
+    uint32_t {$info.stringLocalName}_len = strlen((const char*){$info.name});
+{% if info.stringAllocSize != info.stringLocalName & "_len"  %}
+
+    erpc_assert({$info.stringLocalName}_len <= {$info.stringAllocSize});
+
```

Comment:
move one empty line outside of condition

Reply:
agree